mne/transforms.py

```diff
@@ -905,9 +905,7 @@ class _TPSWarp(object):
 
     References
     ----------
-    .. [1] Bookstein, F. L. "Principal Warps: Thin Plate Splines and the
-           Decomposition of Deformations." IEEE Trans. Pattern Anal. Mach.
-           Intell. 11, 567-585, 1989.
+    ..footbibliography::
```

Comment:
I think this won't do anything without a corresponding `footcite` a few lines above.  Probably should change the notes section to 
```
Based on the method by :footcite:`Bookstein1989` and adaped from code by ....
```

and then make sure that the Bookstein reference is actually in our `doc/references.bib` file.

Reply:
you are also missing a space here:

```suggestion
    .. footbibliography::
```